src/AppBundle/Controller/Report/BankAccountController.php

```diff
@@ -211,9 +211,24 @@ public function deleteAction(Request $request, $reportId, $accountId)
                 'notice',
                 'Bank account deleted'
             );
+        } catch (RestClientException $e) {
```

Comment:
just for curiosity, what happens if the API fails for another RestClientException  (not a BusinessRulesException one ?

Reply:
Tested, nothing happens on frontend. Log entry made. Ive rethrown the exception if data is not set. Handled in same way as any other RestClientException.
I cant catch the BusinessRuleException as all exceptions are converted to RestClientException. Does this need to change?